src/workloads/query/CollScanSimplifiablePredicateSmall.yml

```diff
@@ -19,6 +19,7 @@ AutoRun:
       - standalone-all-feature-flags
       - standalone-classic-query-engine
       - standalone-heuristic-bonsai
+      - standalone-query-stats
```

Comment:
Looks like this workload's auto-run is configured to only run on v7.3 and above. Was it added in v7.3? We should either remove our config from this list or change the branch name, if it's possible to do that. Looks like this applies to all the other `CollScanSimplifiablePredicate*` workloads as well.

Reply:
I didn't notice that. The collscan benchmarks appear to be added in 7.3, I'll ask the author for the proper course of action.

Reply:
@henrinikku said it's ok to mark it for v7.0, done